src/components/ActionCardGrid.tsx

```diff
@@ -4,20 +4,22 @@ import ActionCard, { LocationIcon } from '@site/src/components/ActionCard';
 
 interface ActionCardGridProps {
   title: string;
+  id?: string;
   sections: {
-    title: string;
+    title?: string;
```

Comment:
I dont think the UI supports it being optional right now. I'd change the line below to support that if you want that and make sure to test it.